app/components/account/OrderStateBadge.tsx

```diff
@@ -0,0 +1,77 @@
+// Copied from https://www.vendure.io/docs/typescript-api/orders/order-state/
+type OrderState = | 'Created'
+    | 'Draft'
+    | 'AddingItems'
+    | 'ArrangingPayment'
+    | 'PaymentAuthorized'
+    | 'PaymentSettled'
+    | 'PartiallyShipped'
+    | 'Shipped'
+    | 'PartiallyDelivered'
+    | 'Delivered'
+    | 'Modifying'
+    | 'ArrangingAdditionalPayment'
+    | 'Cancelled'
+// | keyof CustomOrderStates
```

Comment:
These comments here including to the dark mode styling should be entirely removed I think. A Tailwind dark mode will most likely never be part of the starter as this means a full styling overhaul of nearly every component when there's other things to address. It seems a bit out of place including them.

Reply:
I agree - removed.